packages/fxa-settings/src/pages/ResetPassword/index.tsx

```diff
@@ -25,31 +20,41 @@ import { InputText } from '../../components/InputText';
 import LinkRememberPassword from '../../components/LinkRememberPassword';
 import WarningMessage from '../../components/WarningMessage';
 import { isEmailValid } from 'fxa-shared/email/helpers';
-import sentryMetrics from 'fxa-shared/lib/sentry';
+import {
+  BeginResetPasswordHandler,
+  BeginResetPasswordResult,
+} from './container';
```

Comment:
Moving these handler declarations into this file seems better to me.

Not doing so creates an bi-driectional dependency between container.tsx and index.tsx, and we should avoid bi-directional dependencies. The container has to reference the component, since it is a wrapper, but I don't think there's any reason the component needs to know about or reference the container.